Entities/Common/Building/BlobPlacement.as

```diff
@@ -78,6 +78,17 @@ bool serverBlobCheck(CBlob@ blob, CBlob@ blobToPlace, Vec2f cursorPos)
 			return false;
 	}
 
+	CBlob @blobAtPos = map.getBlobAtPosition(cursorPos + Vec2f(1, 1));
+
+	// Are we trying to place a blob on a door/ladder/platform/bridge (usually due to lag)?
+	if (blobAtPos !is null && (
```

Comment:
this should probably check for whether there is any solid tile/blob tile instead of this?

Reply:
fixed, added a function to placementcommon.as